scripts/generate-iot-credential.sh

```diff
@@ -65,43 +67,23 @@ cat > iot-policy-document.json <<EOF
    "Version":"2012-10-17",
    "Statement":[
       {
-	 "Effect":"Allow",
-	 "Action":[
-	    "iot:Connect"
-	 ],
-	 "Resource":"$(jq --raw-output '.roleAliasArn' iot-role-alias.json)"
+     "Effect":"Allow",
+     "Action":[
+        "iot:Connect"
+     ],
+     "Resource":"$(jq --raw-output '.roleAliasArn' iot-role-alias.json)"
  },
       {
-	 "Effect":"Allow",
-	 "Action":[
-	    "iot:AssumeRoleWithCertificate"
-	 ],
-	 "Resource":"$(jq --raw-output '.roleAliasArn' iot-role-alias.json)"
+     "Effect":"Allow",
+     "Action":[
+        "iot:AssumeRoleWithCertificate"
+     ],
+     "Resource":"$(jq --raw-output '.roleAliasArn' iot-role-alias.json)"
  }
    ]
 }
 EOF
 # Now you can create the policy that will enable IoT to assume role with the certificate (once it is attached) using the role alias.
-aws --profile default iot create-policy --policy-name $iotPolicyName --policy-document 'file://iot-policy-document.json' 
-
-# Step 3: Create and Configure the X.509 Certificate
-# Create the certificate to which you must attach the policy for IoT that you created above.
-aws --profile default  iot create-keys-and-certificate --set-as-active --certificate-pem-outfile $iotCert --public-key-outfile $iotPublicKey --private-key-outfile $iotPrivateKey > certificate
-# Attach the policy for IoT (KvsCameraIoTPolicy created above) to this certificate.
-aws --profile default  iot attach-policy --policy-name $iotPolicyName --target $(jq --raw-output '.certificateArn' certificate)
-# Attach your IoT thing (kvs_example_camera_stream) to the certificate you just created:
-aws --profile default  iot attach-thing-principal --thing-name $thingName --principal $(jq --raw-output '.certificateArn' certificate)
-# In order to authorize requests through the IoT credentials provider, you need the IoT credentials endpoint which is unique to your AWS account ID. You can use the following command to get the IoT credentials endpoint.
-aws --profile default  iot describe-endpoint --endpoint-type iot:CredentialProvider --output text > iot-credential-provider.txt
-# In addition to the X.509 cerficiate created above, you must also have a CA certificate to establish trust with the back-end service through TLS. You can get the CA certificate using the following command:
-curl --silent 'https://www.amazontrust.com/repository/SFSRootCAG2.pem' --output cacert.pem
-
-
-export AWS_IOT_CORE_CREDENTIAL_ENDPOINT=$(cat iot-credential-provider.txt)
-export AWS_IOT_CORE_CERT=$(pwd)"/"$iotCert
-export AWS_IOT_CORE_PRIVATE_KEY=$(pwd)"/"$iotPrivateKey
-export AWS_IOT_CORE_ROLE_ALIAS=$iotRoleAlias
-export AWS_IOT_CORE_THING_NAME=$thingName
-        
+aws --profile default iot create-policy --policy-name $iotPolicyName --policy-document 'file://iot-policy-document.json'
```

Comment:
The following lines are commented out but present in this file in the demos repository:
```
# Step 3: Create and Configure the X.509 Certificate
# Create the certificate to which you must attach the policy for IoT that you created above.
# aws --profile default  iot create-keys-and-certificate --set-as-active --certificate-pem-outfile $iotCert --public-key-outfile $iotPublicKey --private-key-outfile $iotPrivateKey > certificate
# # Attach the policy for IoT (KvsCameraIoTPolicy created above) to this certificate.
# aws --profile default  iot attach-policy --policy-name $iotPolicyName --target $(jq --raw-output '.certificateArn' certificate)
# # Attach your IoT thing (kvs_example_camera_stream) to the certificate you just created:
# aws --profile default  iot attach-thing-principal --thing-name $thingName --principal $(jq --raw-output '.certificateArn' certificate)
# # In order to authorize requests through the IoT credentials provider, you need the IoT credentials endpoint which is unique to your AWS account ID. You can use the following command to get the IoT credentials endpoint.
# aws --profile default  iot describe-endpoint --endpoint-type iot:CredentialProvider --output text > iot-credential-provider.txt
# # In addition to the X.509 cerficiate created above, you must also have a CA certificate to establish trust with the back-end service through TLS. You can get the CA certificate using the following command:
# curl --silent 'https://www.amazontrust.com/repository/SFSRootCAG2.pem' --output cacert.pem
```
Is there a reason we excluded it from here?

Reply:
Moved to a new file (cert_setup.sh). Retained these as comments during dev and never removed it.